jetson/percep/pcl.cpp

```diff
@@ -376,8 +280,8 @@ bool PCL::CheckPath(std::vector<std::vector<int>> interest_points,
     //Iterate through interest points
     for(auto cluster : interest_points) {
         for (auto index : cluster) {
-            if(leftLine(pt_cloud_ptr->points[index].z,PCL::pt_cloud_ptr->points[index].x) >= 0  && 
-               rightLine(pt_cloud_ptr->points[index].z,PCL::pt_cloud_ptr->points[index].x) <=0){
+            if(leftLine(pt_cloud_ptr->points[index].x, pt_cloud_ptr->points[index].z) >= 0  && 
```

Comment:
This is a pretty complicated line of code. I don't intuitively know what `leftLine`'s `()` operator would do. Also, a comment would probably be good here.

Reply:
Added comments